modules/end-user-guide/pages/mounting-configmaps-and-secrets.adoc

```diff
@@ -0,0 +1,106 @@
+:navtitle: Mounting ConfigMaps and Secrets
+:keywords: user-guide, configuring, user, configmaps, configmap, secret, secrets, mounting, mount
+:page-aliases:
+
+[id="mounting-configmaps-and-secrets_{context}"]
+= Mounting ConfigMaps and Secrets
+
+ConfigMaps are {platforms-name} objects that store non-confidential data such as configuration values for an application.
+
+Secrets are {platforms-name} objects that store confidential data such as user names, passwords, authentication tokens, and configurations in an encrypted form.
+
+This section describes how to automatically mount {orch-name} ConfigMaps and Secrets to all {devworkspace} containers, creating permanent mount points for components such as:
+
+* Maven configuration, such as the `settings.xml` file 
+* SSH key pairs
+* AWS authorization tokens
+* Configuration files
+* Git credentials store files
+
+.Prerequisites
+
+* A running instance of {prod-short}.
+* You have CLI or GUI access to the {orch-name} cluster of your organization's {prod-short} instance.
+* For CLI users: The `{orch-cli}` command line tool is installed in the operating system you are using.
+
+.Procedure
+
+. Determine an existing ConfigMap or Secret in your user {orch-namespace} to mount to all {devworkspace} containers.
```

Comment:
I would say that this step is a prerequisite: "Create a new ConfigMap or Secret or determine an existing one in your user {orch-namespace} to mount to all workspaces containers."